### PR TITLE
fixing a variable name

### PR DESCRIPTION
**Pull request by g-gonzo**

### CODE REVIEW

**Comment by g-gonzo** [comment]
So I added a class, with 12 spacecraft from the Star Wars collection just to get started.. this is just fixing the misspelling of the list variable.

**Comment by g-gonzo** [comment]
I just realized that this needs the using statement for "Using.Collections.Generics;"

**Comment by g-gonzo** [comment]
Okay, I am waiting for adult supervision before I jack it all up.. Maybe we can get together and look at this... ?

File: obj/StarTrekvsStarWars.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
```
Comment (jostut) [inline obj/StarTrekvsStarWars.csproj.nuget.dgspec.json:1]: I think anything under obj can be deleted and not merged. I'll double check with a dev on monday

**Comment by g-gonzo** [comment]
I believe this Pull Request is obsolete..  I think the changes in here are currently in the main build.. due to the baseline we created during the last meeting together ...  so, do we delete or leave it..  What do we do in a case like this?

**Comment by jostut** [comment]
This is your branch. If you feel this is no longer need then yeah sure.

**Comment by g-gonzo** [comment]
No longer needed..

**Pull request closed without merging**